qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -397,7 +474,7 @@ def _test_trace(
         ][0]
         self.assertEqual(len(parsed_spans), expected_number_of_spans)
 
-        self._verify_contents(parsed_spans, expected_counts)
+        self._verify_contents(parsed_spans, expected_counts, False)
```

Comment:
I would recommend not use just False or True as an argument, it might be not clear what they stand for in this context. Please use `is_cancelled=False`